components/Python/statistics/predictionStats/stats.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import print_function
+
+import argparse
+import os
+import boto3
+import uuid
+
+import numpy as np
+import pandas as pd
+
+from parallelm.components import ConnectableComponent
+from parallelm.mlops import mlops as mlops
+from parallelm.mlops.stats.bar_graph import BarGraph
+
+
```

Comment:
PEP8 require max of 2 lines of space.

Reply:
I formatted the entire code to follow PEP8 guidelines.